src/components/views/messages/MImageBody.tsx

```diff
@@ -117,6 +117,17 @@ export default class MImageBody extends React.Component<IBodyProps, IState> {
                 params.fileSize = content.info.size;
             }
 
+            if (this.image.current) {
+                const clientRect = this.image.current.getBoundingClientRect();
```

Comment:
We should have image's `width` and `height` available already, is it necessary to calculate it from the DOM instead? I don't necessarily understand why it's done this way here, so I may just be in the wrong. 😄

Reply:
We should have those available via `const content = this.props.mxEvent.getContent<IMediaEventContent>()`, see `src/components/views/messages/MImageBody.tsx:321`. If that won't work just resolve this comment please. 😄